packages/eslint-plugin/docs/rules/consistent-type-exports.md

```diff
@@ -13,6 +13,35 @@ This rule aims to standardize the use of type exports style across a codebase.
 
 Given a class `Button`, and an interface `ButtonProps`, examples of code:
 
+### ❌ Incorrect
```

Comment:
The `<!--tabs-->` comment before the heading is actually useful. It will turn the headings into tabs. See https://typescript-eslint.io/rules/array-type#array for example.

Reply:
ok! I added <!--tabs--> above examples